.github/workflows/soundness.yml

```diff
@@ -177,3 +185,21 @@ jobs:
         run: |
           apt-get -qq update && apt-get -qq -y install shellcheck
           git ls-files -z '*.sh' | xargs -0 shellcheck
+
+  yaml-lint:
+    name: Lint YAML
+    if: ${{ inputs.yamllint_enabled }}
+    runs-on: ubuntu-latest
+    container:
+      image: ${{ inputs.yamllint_container_image }}
```

Comment:
I don't think we need a container image at all here we can just run on the runner itself